modules/ossm-release-3-0-TP-1.adoc

```diff
@@ -9,10 +9,57 @@ Module included in the following assemblies:
 
 This release of {SMProductName} is Technology Preview.
 
-// Simple table to help clarify for users the component versions they can use with OSSM 3.0 TP1. Likely a more complicated table will be needed for GA. OSSM 3.0 separates all the Operators such as OCP, Istio, Envoy, Kiali, OTEL/Tempo, etc. This is only what is known to work with OSSM 3.0 TP1.
+// Simple table to help clarify for users the component versions they can use with OSSM 3.0 TP1. Likely a more complicated table will be needed for GA. OSSM 3.0 separates all the Operators such as OCP, Istio, Envoy, Kiali, OTEL/Tempo, etc. This is only what is known to work with OSSM 3.0 TP1 and TP2, as of 12/09/2024.
+
+[id="ossm-release-3-0-TP1-component-versions-2_{context}"]
+== Supported component versions for Technology Preview 2
+
+|===
+|Component |Version
+
+|{ocp-product-title}
+|4.14+
```

Comment:
```suggestion
|4.14 and later
```
From IBM Style: Use "earlier" or "later" to refer to multiple releases: https://www.ibm.com/docs/en/ibm-style?topic=references-product-versions#references-to-multiple-releases